lab-yunjoo/app/component/app-edit-gallery-item-form/app-edit-gallery-item-from-directive.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const angular = require('angular');
+angular.module('demoApp')
+.directive('appEditGalleryItemForm', function(){
+  return {
+    restrict: 'E',
+    replace: true,
+    scope: {
+      title: '=',
```

Comment:
It's easier, more extensible, and more resilient to errors to pass the whole item through, not all of its properties individually.